test/fixtures/ansible-config-invalid.yml

```diff
@@ -0,0 +1,3 @@
+# invalid .ansible-lint config file
```

Comment:
Since there's no fast way to track the fixture file to the test where it's used, it's better to bundle it into the test module.

Reply:
No, not a joke. Doing it without cloning the repo is slower. Also, you misunderstood the direction of the tracking. I'm talking about tracking the test function name when seeing the fixture file.